exporter/datadogexporter/utils/tags.go

```diff
@@ -0,0 +1,105 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"strings"
+	"unicode"
+)
+
+// constants for tags
+const (
+	// maximums for tag string lengths
+	MaxTagLength = 200
+)
+
+// NormalizeTag returns a cleaned up, normalized tags. If the tag can't be normalized
```

Comment:
```suggestion
// NormalizeTag returns a cleaned up, normalized tag. If the tag can't be normalized
```

Reply:
Updated the language here to describe the function more accurately.